### PR TITLE
OSDOCS-3302: Updating the TechPreviewNoUpdate list in 4.10

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OSDOCS-3302

Updated the list to match: https://github.com/openshift/api/blob/release-4.10/config/v1/types_feature.go#L113

Preview: [Understanding feature gates](https://deploy-preview-41695--osdocs.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-enabling-features.html)

### CODE REVIEW

**Comment by mburke5678** [comment]
@duanwei33 Thank you for quickly reviewing my other PRs for feature gates. Would you be able to review this PR also? Thank you very much.

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 1]
```diff
@@ -12,18 +12,26 @@ You can activate any of the following feature sets by using the `FeatureGate` CR
 
 * `TechPreviewNoUpgrade`. This featrue set a subset of the current Technology Preview features on test clusters, where you can fully test them, while leaving the features disabled on production clusters. Enabling this feature set cannot be undone and prevents upgrades. This feature set is not recommended on production clusters. The following Technology Preview features are enabled by this feature set:
 
-** Azure Disk CSI Driver Operator. Enables the provisioning of persistent volumes (PVs) by using the Container Storage Interface (CSI) driver for Microsoft Azure Disk Storage.
-** VMware vSphere CSI Driver Operator. Enables the provisioning of persistent volumes (PVs) by using the Container Storage Interface (CSI) VMware vSphere driver for Virtual Machine Disk (VMDK) volumes.
-** CSI automatic migration. Enables the automatic migration of supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers. Available as a Technology Preview for:
+** Persistent storage using the Alibaba AliCloud Disk CSI Driver Operator. Enables provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk Storage.
```
Comment (duanwei33) [inline modules/nodes-cluster-enabling-features-about.adoc:15]: This should be removed, here is a typo? that you mentioned both Alibaba AliCloud Disk and Microsoft Azure Disk, but anyway, both of them need to be enabled by feature gate in 4.10. (See https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L113)

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 2]
```diff
@@ -12,18 +12,26 @@ You can activate any of the following feature sets by using the `FeatureGate` CR
...
+** Persistent storage using the Microsoft Azure File CSI Driver Operator. Enables provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure File Storage. ???
```
Comment (duanwei33) [inline modules/nodes-cluster-enabling-features-about.adoc:16]: This is correct for Azure File CSI Driver Operator, we can remove the "???"

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 3]
```diff
@@ -12,18 +12,26 @@ You can activate any of the following feature sets by using the `FeatureGate` CR
...
+** VMware vSphere CSI Driver Operator. Enables provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) VMware vSphere driver for Virtual Machine Disk (VMDK) volumes.
```
Comment (duanwei33) [inline modules/nodes-cluster-enabling-features-about.adoc:17]: This should be removed, vSphere CSI Driver doesn't need to be enabled by feature gate in 4.10. (See https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L113)

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 4]
```diff
@@ -12,18 +12,26 @@ You can activate any of the following feature sets by using the `FeatureGate` CR
...
+** CSI automatic migration. Enables automatic migration for supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers. Supported for:
 *** Amazon Web Services (AWS) Elastic Block Storage (EBS)
 *** OpenStack Cinder
+*** Google Compute Engine Persistent Disk (in-tree)
```
Comment (duanwei33) [inline modules/nodes-cluster-enabling-features-about.adoc:21]: Please remove this one.

File: nodes/clusters/nodes-cluster-enabling-features.adoc [hunk 5]
```diff
@@ -13,12 +13,16 @@ include::modules/nodes-cluster-enabling-features-about.adoc[leveloffset=+1]
 .Additional resources
 
 * For more information on the features activated by the `TechPreviewNoUpdate` feature gate, see the following topics:
-
-** xref:../../storage/container_storage_interface/persistent-storage-csi-azure.adoc#persistent-storage-csi-azure-disk[Azure Disk CSI Driver Operator]
+** xref:../../storage/container_storage_interface/persistent-storage-csi-azure.adoc#persistent-storage-csi-azure-disk[Persistent storage using the Alibaba AliCloud Disk CSI Driver Operator]
```
Comment (duanwei33) [inline nodes/clusters/nodes-cluster-enabling-features.adoc:16]: Please remove this.

File: nodes/clusters/nodes-cluster-enabling-features.adoc [hunk 6]
```diff
@@ -13,12 +13,16 @@ include::modules/nodes-cluster-enabling-features-about.adoc[leveloffset=+1]
...
+** [Persistent storage using the Microsoft Azure File CSI Driver Operator] storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file
```
Comment (duanwei33) [inline nodes/clusters/nodes-cluster-enabling-features.adoc:17]: The format looks not correct in preview doc.

**Review by duanwei33 — COMMENTED** [review]
@jsafrane Could you help take a look as well?

File: modules/nodes-cluster-enabling-features-about.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/nodes-cluster-enabling-features-about.adoc:15]: @duanwei33 is Alibaba in the feature set? I don't see it at https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L113

**Comment by mburke5678** [comment]
@duanwei33 I apologize. Apparently, this PR wasn't ready for your review. Thank you for your thoughtful comments.

File: modules/nodes-cluster-enabling-features-about.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/nodes-cluster-enabling-features-about.adoc]: @duanwei33 Can you help with this ^^?

Comment (duanwei33) [reply modules/nodes-cluster-enabling-features-about.adoc]: @mburke5678 
Really sorry for my typo, none of them need to be enabled by feature gate in 4.10. So this line should be removed.

**Comment by mburke5678** [comment]
@duanwei33 PTAL

**Comment by duanwei33** [comment]
@mburke5678 Thanks for the update.
It looks good to me for stoarge part.

**Comment by mburke5678** [comment]
@lpettyjo @rolfedh Can I ask you to PTAL at the parts of the PR that apply to your areas? I believe most of the features enabled by this feature set are Storage and Build. I want to make sure I have described the features properly and the links to the docs in the Additional Resources are the best places to link to.

**Review by rolfedh — APPROVED** [review]
Looks good to me, @mburke5678 ! Thanks.

**Comment by jsafrane** [comment]
lgtm from storage side

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 7]
```diff
@@ -8,22 +8,27 @@
...
+The following Technology Preview features are enabled by this feature set:
++
+** Microsoft Azure File CSI Driver Operator. Enables the provisioning of persistent volumes (PVs) by using the Container Storage Interface (CSI) driver for Microsoft Azure File Storage.
+** CSI automatic migration. Enables automatic migration for supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers. Supported for:
 *** Amazon Web Services (AWS) Elastic Block Storage (EBS)
 *** OpenStack Cinder
 *** Azure Disk
-*** Google Cloud Platform Persistent Disk.
+*** Azure File
+*** Google Cloud Platform Persistent Disk (CSI).
```
Comment (skrthomas) [inline modules/nodes-cluster-enabling-features-about.adoc:23]: I think you can remove the period here

File: nodes/clusters/nodes-cluster-enabling-features.adoc [hunk 8]
```diff
@@ -13,12 +13,11 @@ include::modules/nodes-cluster-enabling-features-about.adoc[leveloffset=+1]
...
-** xref:../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-vsphere[VMware vSphere CSI Driver Operator]
+** xref:../../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file[Azure File CSI Driver Operator]
 ** xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration]
 ** xref:../../operators/operator-reference.html#cluster-cloud-controller-manager-operator_platform-operators-ref[Cluster Cloud Controller Manager Operator]
-** xref:../../support/remote_health_monitoring/insights-operator-simple-access.adoc#insights-operator-simple-access[Importing simple content access certificates with Insights Operator ]
+** xref:../../cicd/builds/build-strategies.html#builds-using-build-volumes_build-strategies-s2i[Source-to-image (S2I) build volumes] and xref:../../cicd/builds/build-strategies.html#builds-using-build-volumes_build-strategies-docker[Docker build volumes].
```
Comment (skrthomas) [inline nodes/clusters/nodes-cluster-enabling-features.adoc:19]: I dont think you need the period here. It is a longer list item but still not a complete sentence

**Review by skrthomas — COMMENTED** [review]
2 small comments, looks great otherwise :)

**Pull request merged**